src/dropdown/dropdown.ts

```diff
@@ -1,11 +1,24 @@
-import {Directive, Input, HostListener} from '@angular/core';
+import {Directive, ElementRef, Input, HostListener} from '@angular/core';
```

Comment:
Just a note that I changed this file so I could do some testing as the dropdown does not close when you click on a menu item nor does it close when you click outside the menu on the rest of the document.  Feel free to reject the PR if this is totally unwanted in the library and I will remove.  Just know the menu UX is very poor in the interim.